src/commands/run.zsh

```diff
@@ -190,14 +192,20 @@ function _zunit_execute_test() {
 
       return
     elif [[ $state -eq 78 ]]; then
-      _zunit_error "Test took too long to run. Terminated after $time_limit seconds" $output
+      _zunit_error $output
 
       return
     elif [[ -z $allow_risky && $state -eq 248 ]]; then
+      # If --verbose is specified, print test output to screen
+      [[ -n $verbose && -n $output ]] && echo $output
```

Comment:
Yeah it should do, stderr is redirected to stdout [here](https://github.com/zunit-zsh/zunit/blob/next/src/commands/run.zsh#L182)